shell/monitor/monitor.sh

```diff
@@ -34,9 +39,71 @@ _monitor_set_experiment_info_func(){
   _monitor_experiment_info_func=( "$@" )
 }
 
+_monitor_login(){
+
+  local output code
+
+  rm -f "$_monitor_cookie_file"
+
+  output=$(curl -sS -w '\n%{http_code}\n' -XPOST https://${_monitor_eshost}/auth/log_in -H 'Content-type: application/json' \
+                -c "$_monitor_cookie_file" -b "$_monitor_cookie_file" -d '{"email": "'"$_monitor_proxy_email"'", "password": "'"$_monitor_proxy_password"'" }')
+
+  code=$( echo "$output" | tail -n1)
```

Comment:
Minor whitespace thing here, and maybe `_monitor_get_code` can be made into a function since we do it in a few places (we could also put `"$(_monitor_get_code "$output")"` into the test below directly if we only read it once in this function).